src/main/java/org/candlepin/subscriptions/resteasy/ResteasyConfiguration.java

```diff
@@ -53,11 +52,4 @@ public void addViewControllers(ViewControllerRegistry registry) {
     registry.addViewController("/api-docs").setViewName("redirect:/api-docs/index.html");
     registry.addViewController("/api-docs/").setViewName("redirect:/api-docs/index.html");
   }
```

Comment:
I think this whole method can go too.

Reply:
I remove this method, but when I tested it it resulted in a 404 unless you use the full URL path to index.html

Reply:
Okay, pay me no mind then!